migration_scripts/extract_text_embeddings.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
If this is intended to be executed multiple times, could we make it a standalone .py file that acts as a CLI tool with arguments? this could even go into the graphrag source code.